dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/controller/GaenController.java

```diff
@@ -106,9 +105,7 @@ public GaenController(GAENDataService dataService, FakeKeyService fakeKeyService
 			if (!validationUtils.isValidBase64Key(key.getKeyData())) {
 				return () -> new ResponseEntity<>("No valid base64 key", HttpStatus.BAD_REQUEST);
 			}
-			if (this.validateRequest.isFakeRequest(principal, key)) {
-				continue;
-			} else {
+			if (!this.validateRequest.isFakeRequest(principal, key)) {
```

Comment:
I would rather not use logical nots if not needed. Fair enough the else is unnecessary though.

Reply:
I got to admit that I was expecting that comment, but hoped I could get around it :( Thanks for insisting...